torch/ao/quantization/pt2e/eval_utils.py

```diff
@@ -17,6 +17,10 @@ def _replace_dropout_for_eval(m: torch.fx.GraphModule):
     # Avoid circular dependencies
     from .utils import get_aten_graph_module
 
+    # Needed to ensure subgraph matches are self-contained
+    m.graph.eliminate_dead_code()
+    m.recompile()
```

Comment:
is there a test catching this?

Reply:
Yeah this is tested in the existing `test_move_exported_model_to_eval`